### PR TITLE
RUMM-2332 WebView tracking through message bus

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

Send WebView events through the message bus.

### How?

Use the `FeatureMessageAttributes.AnyEncodable` type-erasure wrapper for passing `Encodable` objects through the bus. This wrapper is still inner to `FeatureMessageAttributes` but could be generalised if needed.

Logs and RUM are now able to read events from the bus and forward it to their events writers.

### Review checklist
- [x] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes

### Custom CI job configuration (optional)
- [x] Run unit tests
- [x] Run integration tests
- [ ] Run smoke tests

### CODE REVIEW

File: Tests/DatadogTests/Datadog/RUM/WebView/WebRUMEventConsumerTests.swift [hunk 1]
```diff
@@ -65,7 +65,7 @@ class WebRUMEventConsumerTests: XCTestCase {
 
     func testWhenValidWebRUMEventPassedWithoutRUMContext_itPassesToWriter() throws {
```
Comment (ncreated) [inline Tests/DatadogTests/Datadog/RUM/WebView/WebRUMEventConsumerTests.swift:66]: convention/ All tests for this (and Logs) consumer are named with _"itPassesToWriter"_, which isn't now true. It should now say that respective message is passed to Core Event Bus.

File: Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift [hunk 2]
```diff
@@ -51,22 +51,13 @@ internal class WebEventBridge {
...
-                    event: wrappedEvent,
-                    internalLog: (eventType == Constants.eventTypeInternalLog)
-                )
-            } else {
-                DD.logger.warn("A WebView log is lost because Logging is disabled in the SDK")
-            }
+        if eventType == Constants.eventTypeLog || eventType == Constants.eventTypeInternalLog {
+            try logEventConsumer.consume(
+                event: wrappedEvent,
+                internalLog: (eventType == Constants.eventTypeInternalLog)
```
Comment (ncreated) [inline Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift:57]: This makes me realise that `internalLog` is a leftover after https://github.com/DataDog/dd-sdk-ios/pull/819 and it leaves a dead path in the code, no? There is no use of `internalLog` bool inside this `.consume(event:internalLog:)` method as far as I can see. How about cleaning it up 🏕️?

**Review by ncreated — COMMENTED** [review]
Looks good 👍. Although we don't yet know how WV Tracking will be designed in V2, this is a great step forward in isolating it from `LoggingFeature` and `RUMFeature` 🏅.

I left some feedback on convention and potential cleanup, LTM!

File: Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift (hunk 2 above)
Comment (maxep) [reply Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift:57]: Ah yes, I noticed it but forgot to mention it. Browser SDK also migrated to RUM Telemetry, so we might no longer receive `internal_log` event type. I will double check with them and remove this 👍

**Review by ncreated — APPROVED** [review]
👌🙌

**Pull request merged**